mne/viz/evoked.py

```diff
@@ -1264,12 +1238,6 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
                       verticalalignment='center')
         title_ax.axis('off')
 
-    # prepare axes for topomap
-    # slightly convoluted due to colorbar placement and for vertical alignment
-    ts = len(times) + 2
-    map_ax = [plt.subplot(4, ts, x + 2 + ts) for x in range(ts - 2)]
-    cbar_ax = plt.subplot(4, 3 * (ts + 1), 6 * (ts + 1))
```

Comment:
why?

Reply:
It's in `_prepare_joint_axes` now ... or what do you mean?

Reply:
ok ok. I am always careful when I see red lines :)